controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ControllerMaintenance.java

```diff
@@ -118,13 +118,13 @@ public Intervals(SystemName system) {
             this.outstandingChangeDeployer = duration(3, MINUTES);
             this.versionStatusUpdater = duration(3, MINUTES);
             this.readyJobsTrigger = duration(1, MINUTES);
-            this.deploymentMetricsMaintainer = duration(5, MINUTES);
+            this.deploymentMetricsMaintainer = duration(6, MINUTES);
```

Comment:
```suggestion
            this.deploymentMetricsMaintainer = duration(10, MINUTES);
```
With so many collisions already, and at such a low rate, this one probably needs more bump?